pom.xml

```diff
@@ -419,6 +435,11 @@
                 <artifactId>validation-api</artifactId>
                 <version>2.0.1.Final</version>
             </dependency>
+            <dependency>
+                <groupId>javax.xml.bind</groupId>
+                <artifactId>jaxb-api</artifactId>
+                <version>2.3.0</version>
+            </dependency>
```

Comment:
Need java 11 for opensearch java client, therefore need jaxb-api, jaxb-runtime, jakarta.activation for cxf-xjc-plugin.
https://artofcode.wordpress.com/2019/02/28/generating-classes-from-xsd-under-java-11-the-right-way/

Reply:
I suspect this is not correct.  I suspect `jakarta.xml.bind-api` is the correct dependency (already referenced in dependency management).